pkg/test/echo/server/endpoint/grpc.go

```diff
@@ -113,6 +115,11 @@ func (s *grpcInstance) Start(onReady OnReadyFunc) error {
 	}
 	s.server = s.newServer(opts...)
 
+	// add the standard grpc health check
+	healthServer := health.NewServer()
+	healthServer.SetServingStatus("", grpcHealth.HealthCheckResponse_SERVING)
```

Comment:
should this be set in `onReady`? otherwise it will return `HEALTHY` before it is

Reply:
Yes, fixed